src/main/shared/util/utils.ts

```diff
@@ -0,0 +1,7 @@
+export class Utils {
+   static keyValueObject<T>(key: string, value: T): { [key: string]: T } {
+      const object: { [key: string]: T } = {};
+      object[key] = value;
+      return object;
+   }
```

Comment:
It is just a util to avoid boilerplate code creating a simple object with a key and a value.